2003/kangraemin/app/src/main/java/com/example/kangraemin/model/remote/datadao/MovieImpl.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.kangraemin.model.remote.datadao
+
+import com.example.kangraemin.model.remote.datamodel.Movies
+import io.reactivex.Flowable
+
+class MovieImpl(
+    private val movieApi: MovieInterface
+) : RemoteMovieDataSource {
```

Comment:
네이밍은 일관성 있게 다른 클래스, 인터페이스와 구분할 수 있도록 명확히 해주세요.
```suggestion
class MovieRemoteDataSourceImpl(
    private val movieApi: MovieApi
) : MovieRemoteDataSource {
```

Reply:
수정하였습니다 
4e81ab0